src/components/Link/utils.ts

```diff
@@ -16,7 +16,7 @@ export const linkStyles: ComponentStylePropWithVariants<TextStyle, States> = {
             color: 'colors.onSurfaceDisabled',
             opacity: 0.38,
             // @ts-ignore
-            cursor: 'default',
+            // cursor: 'default',
```

Comment:
wouldn't this remove the cursor?
@thetaungg

Reply:
Thanks, Shashank
Fixed now